packages/components/src/Components/SkeletonTable/SkeletonTable.js

```diff
@@ -19,19 +19,28 @@ class SkeletonTable extends React.Component {
     createRows = () => {
         const { colSize, rowSize, columns, paddingColumnSize } = this.props;
         const numberOfCols = columns ? columns.length : colSize;
-        return this.newArray(rowSize).map(() => ({ cells: this.newArray(paddingColumnSize).map(() => '').concat(
+        return this.newArray(rowSize).map(() => ({ disableSelection: true, cells: this.newArray(paddingColumnSize).map(() => '').concat(
             this.newArray(numberOfCols).map(() => ({ title: <Skeleton size={ SkeletonSize.md } /> }))
         ) }));
     };
 
+    selectVariant = () => {
+        const { isRadio } = this.props;
+
+        return isRadio ? RowSelectVariant.radio : RowSelectVariant.checkbox;
```

Comment:
Unfortunately this is not backwards compatible as older version of PF react table doesn't have `RowSelectVariant` present. We can always fallback to string value, once apps update their version of react-table they'll start using the imported `RowSelectVariant`. With this you don't even have to update the version of react-table in root package.
```suggestion
        return isRadio ? (RowSelectVariant?.radio || 'radio') : (RowSelectVariant?.checkbox || 'checkbox');
```